src/backplanes/backplanes_rings.py

```
@@ -60,28 +61,44 @@ def create_ring_backplanes(snapshot: ObsSnapshot,
     for bp_cfg in rings_cfg:
-        name = bp_cfg['name']
-        method = bp_cfg.get('method', name)
+        bp_name = bp_cfg['name']
+        method = bp_cfg.get('method')
+        if method is None:
+            raise ValueError(f'Ring backplane "method" is required: {bp_name}')
+        units = bp_cfg.get('units')
+        if units is None:
+            raise ValueError(f'Ring backplane "units" is required: {bp_name}')
+
+        logger.debug(f'{closest_planet}: Creating ring backplane {bp_name}')
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)